src/foam/u2/wizard/agents/AlternateFlowAgent.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.wizard.agents',
+  name: 'AlternateFlowAgent',
```

Comment:
@KernelDeimos so we have injecting AlternateFlow as an agent, and also as an AlternateFlowWizardlet indirectly through the capability hierarchy. I think artur is trying to use this to select Anonymous Transfer only for the Transfer Type wizardlet.

Can he also do this by making the top level capability use AlternateFlowWizardlet? Or is using the AlternateFlowWizardlet only applicable for dynamicActions. 

Whereas AlternateFlowAgent will be used for controlling the isAvailable and isVisible of wizardlets in a capability DAG.